isc_export.py

```diff
@@ -0,0 +1,36 @@
+from icalendar import Calendar as iCalendar, Event as iEvent, vText, vDatetime
+import tempfile, os # being able to write a file
+from cal.schema import Event as OurEvent
+from cal import app
+from time import strftime
+from datetime import datetime
+
+def main():
```

Comment:
Can we factor out a function that takes in a list of `OurEvent` and returns the `iCalendar` object?

SQL querying and file IO (we might just serve the string directly w/ flask instead of saving it anywhere) should be separate.